lms/djangoapps/commerce/api/v1/views.py

```diff
@@ -64,10 +73,17 @@ class OrderView(APIView):
     """ Retrieve order details. """
 
     authentication_classes = (SessionAuthentication,)
-    permission_classes = (IsAuthenticated,)
+    permission_classes = (IsAuthenticatedOrActivationOverridden,)
```

Comment:
My concern about dropping the change is that the LMS shoppingcart + receipt page are still there, so technically there could be an Open edX operator who depends on them, and wants to remove the activation requirement...you see where that is going.  So I think we should leave it in.  I will take a look at ways to DRY out the code -- I wanted to do that originally but we really don't have a good utility package hierarchy in place for promoting these types of generalizations.